### ISSUE TITLE
Advanced Tabs fails WCAG Guidelines

### ISSUE THREAD

**Issue by DogwoodJohn** [WPDevelopers/essential-addons-for-elementor-lite#744]
The Advanced Tab element failed accessibility WCAG guidelines for aria-required-parent and list structure.  Since it has role=tab on the li tags, it needs to have role=tablist on the surrounding ul tag.

Wordpress 6.1.1
Elementor 3.12.2
Elementor Pro 3.12.3
EAE 5.7.1

**Comment by IftekharPial** [comment]
Hello @DogwoodJohn 

Sorry for the inconvenience, You will be glad to know that Our Dev team has already fixed the ‘WCAG Guidelines’ issue on EA Advanced Tab and it will be released very soon.

In the meantime, we have a Dev version containing the fix regarding the issue, you can find that here : https://d.pr/f/hXxV3G

Please make sure to make a Backup of your site before any Updates.

Note: Please Regenerate the asset after updating the Dev version
To regenerate assets, you can go to wp-admin -> Essential Addons then, switch to the ‘Tools’ tab and hit the ‘Regenerate Assets’ button. Check out this screenshot: https://d.pr/i/FuLkYy

Hopefully, it will solve your issue
Let me know how it goes
Thanks

**Comment by IftekharPial** [comment]
Hello again @DogwoodJohn 

You will be glad to know that We have released this fix on our  Essential Addons v.5.8.10

Please update to the latest version.

Hopefully, it will solve your issue.
Let me know how it goes.
Thanks
